<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHalaman_Utama">
    <property name="title">
     <string>Halaman Utama</string>
    </property>
    <addaction name="actionData_Client"/>
    <addaction name="actionData_Kasus"/>
    <addaction name="actionData_Arsip"/>
    <addaction name="actionData_Keuangan"/>
   </widget>
   <addaction name="menuHalaman_Utama"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionData_Client">
   <property name="text">
    <string>Data Client</string>
   </property>
  </action>
  <action name="actionData_Kasus">
   <property name="text">
    <string>Data Kasus</string>
   </property>
  </action>
  <action name="actionData_Arsip">
   <property name="text">
    <string>Data Arsip</string>
   </property>
  </action>
  <action name="actionData_Keuangan">
   <property name="text">
    <string>Data Keuangan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
